Editor/EditorCore/EditorMeshUtility.cs

```diff
@@ -108,6 +111,10 @@ public static void Optimize(this ProBuilderMesh mesh, bool generateLightmapUVs =
             if (Experimental.meshesAreAssets)
                 TryCacheMesh(mesh);
 
+            umesh.RecalculateUVDistributionMetric(0);
+            if (uv2Built)
+                umesh.RecalculateUVDistributionMetric(1);
```

Comment:
If I understand this correctly, the minimum UV area to consider value is relative to the texture size, where one texel is equal to `1 / (texture width, height)`. So it looks like this would be effectively ignoring most of the UVs on the mesh.

The default is 1e-9f, why not leave it at that?

Also I'm not sure I understand why it is first called with a parameter of `0`, then if UV2 is built, again with `1`.

Reply:
It actually has two signatures, I'm using this one: https://docs.unity3d.com/ScriptReference/Mesh.RecalculateUVDistributionMetric.html 
The first param is the UV set, the second is the threshold, which I left to default.

Reply:
Addressed compatibility with older editors in [08af8da](https://github.com/Unity-Technologies/com.unity.probuilder/pull/447/commits/08af8daaec54014b7be0b9d920de8265bf24f60a).

Reply:
Cool, I didn't notice the overload. Looks good to me.